### ISSUE TITLE
Question about modern chess engines...

### ISSUE THREAD

**Issue by SLW2K** [jhonnold/berserk#287]
I have a question for you on modern chess programming theory. Why do modern engines place more emphasis on board position than on piece capture? I have a large collection of chess programs & engines (200+), and the older chess programs from the 70's, 80', & 90's tend to place more emphasis on piece capture & pawn promotion, and controlling the game in that manner, where the modern engines place more emphasis on board positions. Stockfish, Komodo, and programs like yours are obviously much more powerful than the best programs from the 90's, but with all the modern processing power and accumulated chess programming knowledge, why not more piece grabbing programs that would obviously excel?

Shawn Weber

**Comment by tissatussa** [comment]
interesting question, i await the reply.
in the meantime i want to add this :

being a chess player, i'm beginning to understand how chess should be played best .. a major rule is that captures are not forced so eg. you can do in-between-moves .. and in general you should place your pieces so you can get advantage by proper recapture .. avoid trading, only trade when needed or to get advantage, better let the opponent pieces be misplaced. This kind of "massage" by non-capturing seems best .. and sacrificing material is good when it brings initiative and attack.

**Comment by jhonnold** [comment]
> why not more piece grabbing programs that would obviously excel?
Why do you assume they would excel? 

TLDR; SF and Komodo are positional because they learned to be, not because authors chose this path.

Early in Berserk's creation I was choosing how important certain weights were in it's evaluation. I easily could have influenced the engine to believe piece grabbing was the best option. Early play by this engine did tend to grab b2/g2 pawns with it's queen (poisoned pawns).

However, this is far from optimal and [texel tuning](https://www.chessprogramming.org/Texel's_Tuning_Method) allows an engine to tune it's weights to values that **simply win games**. If this results in non-materialistic engines, then that is best.

**Comment by SLW2K** [comment]
Thanks for the reply. What I meant by "they would excel" is they would excel because of current technology and accumulated programming knowledge compared to 20 years ago, not because it's "the best way". I'm personally rated about 2300, and I tend to play a materialistic game as taking more pieces has scored more wins for me than positional play (which I don't care for - I don't like positional games where you sacrifice piece after piece to win). Anyway, that's why I posted this, thanks for the reply.

Shawn Weber
